README.md

```diff
@@ -9,10 +9,12 @@ ebpfpub is a generic function tracing library for Linux that supports tracepoint
 ## Building
 
 ### Prerequisites
-* A recent Clang/LLVM installation (8.0 or better), compiled with BPF support
+* A recent Clang/LLVM installation (8.0 or better), compiled with BPF support.
```

Comment:
We could add a note here to specify that the preferred osquery-toolchain build method already comes with the LLVM libraries